doorman/settings.py

```diff
@@ -68,12 +88,18 @@ class DevConfig(Config):
 
 
 class TestConfig(Config):
+    """
+    This class specifies a configuration that is used for our tests.
+    """
     TESTING = True
     DEBUG = True
-    # SQLALCHEMY_DATABASE_URI = 'sqlite://'
-    # SQLALCHEMY_DATABASE_URI = 'postgresql://localhost:6432/doorman_test'
+
+    BROKER_URL = 'redis://localhost:6379/0'
+    CELERY_RESULT_BACKEND = 'redis://localhost:6379/0'
     SQLALCHEMY_DATABASE_URI = 'postgresql://localhost:5432/doorman_test'
+
     WTF_CSRF_ENABLED = False
+    CELERY_ALWAYS_EAGER = True      # Don't use the Celery queue by default
```

Comment:
The three tests you mention above appear to be failing because of this setting. Remove it, and tests will begin to pass again.